.github/workflows/ci.yaml

```
@@ -331,7 +331,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: scbexperimental/test-scan
+          repository: securecodebox/test-scan
           path: ./scanners/test-scan/scanner/
           # Note: not prefixed with a "v" as this seems to match nmap versioning standards
           tags: "latest"
```

This needs to be configured to use the commit sha and ref to properly work.
Otherwise this will just build the latest tag, which doesn't work with the helm install config for the test-scan in the integration test stage.

```suggestion
          tag_with_ref: true
          tag_with_sha: true
```